gravitee-node-certificates/src/main/java/io/gravitee/node/certificates/AbstractKeyStoreLoaderManager.java

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.node.certificates;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.gravitee.common.util.KeyStoreUtils;
+import io.gravitee.node.api.certificate.*;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableEntryException;
+import java.security.cert.CertificateException;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Predicate;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Slf4j
+public class AbstractKeyStoreLoaderManager {
+
+    private final Map<String, KeyStoreLoader> loaders;
+    private final KeyStoreLoader platformKeyStoreLoader;
+    protected final RefreshableX509Manager refreshableX509Manager;
+    private final String serverId;
```

Comment:
I agree it is not bounded to a serverId.
It is very helpful for logging.
Maybe I can pass along something called loggingContext/Prefix containing `"server [tcp]"` this way you actually need a server id but it will appear in the logs for debug.